gui-wm/labwc/labwc-0.7.4.ebuild

```diff
@@ -24,7 +24,7 @@ RDEPEND="
 	dev-libs/glib:2
 	>=dev-libs/libinput-1.14
 	dev-libs/libxml2:2
-	>=gui-libs/wlroots-0.18.0:=[X?]
+	gui-libs/wlroots:0/17[X?]
```

Comment:
```suggestion
	gui-libs/wlroots:0/17=[X?]
```

Reply:
Adding in the `=` leads to the following error message:

```
Error(s) in metadata for 'gui-wm/labwc-0.7.4':
  DEPEND: Improper context for slot-operator "built" atom syntax: gui-libs/wlroots:0/17=[X?]
  RDEPEND: Improper context for slot-operator "built" atom syntax: gui-libs/wlroots:0/17=[X?]
```

without it, the build continues as normal.

Reply:
Right, `=` means that any slot or sub-slot is okay, but `0/17` already fully specified the slot/sub-slot pair. See [this part](https://devmanual.gentoo.org/general-concepts/dependencies/#slot-dependencies) of the Gentoo dev guide.

Reply:
Ooh. Thanks for pointing it out.